cfpb/consumerfinance.gov
# Publish cf.gov main Docker Image to Github

trying out thee github example workflow to see it it works. Testing on this branch first.



## Additions

-New yml file in the GitHub workflows that should build and publish our root docker imag, and tiger on pushes to a single branch (in this test its thee branch itself but for final push it should be main)



## How to test this PR

1.There should be a new action called "Publish cf.gov Docker Image", this should make a new docker image package for us to store in GitHub


## Screenshots


## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
